### ISSUE TITLE
Translate to Simplified Chinese (zh-CN) 

### ISSUE THREAD

**Issue by innovation64** [huggingface/cookbook#34]
Hi ! I'm doing some RAG research, and I just found your guys' post on this cookbook tutorial. it was awesome. I want to translate it into Chinese. I'm not sure it is ok.

**Comment by MKhalusova** [comment]
Yes! That would be great! I will add instructions on how to add translations later this week, but it should be similar to how we translate courses and docs - [see here](https://github.com/huggingface/course/blob/main/README.md)

**Comment by innovation64** [comment]
Hi there 👋

Let's translate the course to Simplified Chinese so that the whole community can benefit from this resource 🌎!

Below are the chapters and files that need translating - let us know here if you'd like to translate any and we'll add your name to the list. Once you're finished, open a pull request and tag this issue by including `#issue-number` in the description, where `issue-number` is the number of this issue.

>🙋 If you'd like others to help you with the translation, you can also post in our [forums](https://discuss.huggingface.co/c/course/20) or tag [@_lewtun](https://twitter.com/_lewtun) on Twitter to gain some visibility.

We've also created a `chinese-simplified-translations` channel on the Hugging Face Discord in case you wish to discuss translation details there. Just follow the instructions here 👉 https://discord.gg/hKnxnxUr

# Chapters
----
## Open-Source AI Cookbook
- [ ] Automatic Embeddings with TEI through Inference Endpoints
- [x] Embedding multimodal data for similarity search
- [ ] Fine-tuning a Code LLM on Custom Code on a single GPU
- [x] Simple RAG using Hugging Face Zephyr and LangChain
- [ ] Advanced RAG on HuggingFace documentation using LangChain
- [ ] RAG Evaluation

**Comment by innovation64** [comment]
@MKhalusova  hi , I just found there is a bug in merged #40, they can't pass the check lowercase, Should I drop out the zh-CN. Change it just zh? or just ignore it?
![image](https://github.com/huggingface/cookbook/assets/45715979/abf827c3-1636-4cdb-bfd2-0696beb81e09)

**Comment by MKhalusova** [comment]
@innovation64 Just ignore it, it's a bug in the notebook filename checker (it should ignore the language prefix). The content was built correctly, and will fix this check soon.
